docs/solana/options.rst

```diff
@@ -1,5 +1,202 @@
+.. _solana_inlining:
+--solana_inlining
+~~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a list of paths to inlining files for Solana contracts. These files are parsed and used to prove properties.
+
+**When to use it?**
+
+TODO: @Jorge please advise.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --solana_inlining <path_to_inlining_file>
+
+.. _solana_summaries:
+--solana_summaries
+~~~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a list of paths to summary files for Solana contracts. These files are parsed and used to prove properties.
+
+**When to use it?**
+
+TODO: @Jorge please advise.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --solana_summaries <path_to_summaries_file>
+
+.. _cargo_features:
+--cargo_features
+~~~~~~~~~~~~~~~~~
+
+**What does it do?**
+
+Provides the prover with a whitespace-separated list of extra `Cargo features <https://doc.rust-lang.org/cargo/reference/features.html>` passed to the build script.
+These features are then passed to ``cargo`` to build the project.
+
+**When to use it?**
+
+Use it when there is a need to enable a specific `Cargo feature <https://doc.rust-lang.org/cargo/reference/features.html>` to compile the source code.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --cargo_features <feature_1> <feature_2>
+
+.. _msg:
+--msg
+~~~~~
+
+**What does it do?**
+
+Adds a description message to your run, similar to a commit message. This message appears on the Prover dashboard and in the rule report.
+Note that you need to wrap your message in quotes if it contains spaces.
+
+**When to use it?**
+
+Adding a message makes it easier to track several runs. It is very useful if you are running many verifications simultaneously.
+It is also helpful to keep track of a single file verification status over time, so we recommend always providing an informative message.
+
+**Example**
+
+.. code-block:: bash
+
+    certoraSolanaProver --build_script <path_to_build_script> --msg 'Removed an assertion'
+
+.. _rule:
+--rule
+~~~~~~
+
+**What does it do?**
+
+Formally verifies one or more specified properties instead of the whole specification file. Can also verify an invariant.
+
+**When to use it?**
+
+This option saves a lot of run time. Use it whenever you care about only a
+specific subset of a specification's properties. The most common case is when
+you add a new rule to an existing specification. The other is when code changes
+cause a specific rule to fail; in the process of fixing the code, updating the
+rule, and understanding counterexamples, you likely want to verify only that
+specific rule.
+
+**Example**
+
+If `Bank.rs` includes the following:
+
+.. code-block:: text
+
+    invariant address_zero_cannot_become_an_account()
+    rule withdraw_succeeds()
+    rule withdraw_fails()
```

Comment:
these seem to be examples from CVL. In CVLR they would be `rule_withdraw_succeeds` and the full name, inluding `rule_` should be used as a rule name.